modules/mobile/app/controllers/mobile/v0/appointments_controller.rb

```diff
@@ -46,9 +46,16 @@ def cancel
       end
 
       def create
+        Rails.logger.info('mobile appointments create', user_uuid: @current_user.uuid,
+                          params: params.except(:description,
```

Comment:
@jperk51 added in some logging and removed any potential PII fields. What qualifies as PII? name, address, phone number etc are obvious but would knowing the exact time and place of a users appointment qualify?

Reply:
I wouldn't think so, but I can confirm.

Reply:
DSVA support confirmed it shouldn't be a problem.